service/ziti-tunnel/dto/dtos.go

```diff
@@ -200,3 +200,10 @@ type MfaChallenge struct {
 
 type MfaResponse struct {
 }
+
+type ControllerEvent struct {
+	ActionEvent
+	Fingerprint 		string
+	ControllerVersion	string
+	IdentityName		string
```

Comment:
IdentityName is not needed in this struct nor is the ControllerVersion. This should just be Fingerprint

Reply:
removed the unnecessary fields